proposals/params-span.md

```diff
@@ -51,8 +51,9 @@ Two overloads cannot differ by `params` modifier alone.
 
 `params` parameters are marked in metadata with a `System.ParamArrayAttribute`.
 
-A `params ReadOnlySpan<T>` is implicitly `scoped` unless explicitly annotated with `[UnscopedRef]`.
-Within the `params` method, the compiler will use escape analysis to report diagnostics if the `scoped` span is captured or returned.
+A `params ReadOnlySpan<T>` is implicitly `scoped`, and can be declared `scoped` explicitly.
```

Comment:
I didn't see anything in the notes or comments on this PR to indicate why we are making this change. Since we are disallowing explicitly marking it UnscopedRef, I think we should disallow marking it scoped here. No point and people will be falsely lead to think there is a difference.

Reply:
Yes, allowing explicit `params scoped` is consistent with `scoped out`.